.github/workflows/build_oracle8.yml

```diff
@@ -101,51 +109,44 @@ jobs:
           os: Oracle8
           os-full-name: OracleServer-8.9
 
-      - name: Compile Boost
-        uses: ./.github/workflows/compile-boost
-        with:
-          prefix: "../rte-antares-deps-Release/"
       - name: Compile TBB
         uses: ./.github/workflows/compile-tbb
 
-      - name: Install dependencies
+      - run: |
+          mkdir -p ${{ github.workspace }}/vcpkg_cache
+
+      - name: vcpkg install
         run: |
-          source /opt/rh/gcc-toolset-10/enable
-          pip3 install wheel #Too late to install in requirements.txt
-          pip3 install -r requirements-tests.txt
+          pushd vcpkg
+          git fetch --unshallow
+          ./bootstrap-vcpkg.sh
+          popd
+          vcpkg/vcpkg install
 
       - name: Configure
         run: |
           [[ ${{ matrix.xprs.value }} == "XPRESS-ON" ]] && XPRESS_VALUE="ON" || XPRESS_VALUE="OFF" 
```

Comment:
may be delete xpress matrix

Reply:
I open a new issue to track this : https://github.com/AntaresSimulatorTeam/antares-xpansion/issues/781